src/flashmd/ipi.py

```diff
@@ -109,11 +109,11 @@ def flashmd_vv(motion):
 
         if rescale_energy:
             info("@flashmd: Energy rescale", verbosity.debug)
-            new_energy = sim.properties("potential") + sim.properties("kinetic_md")
+            new_energy = sim.properties("conserved")
             kinetic_energy = sim.properties("kinetic_md")
             alpha = np.sqrt(1.0 - (new_energy - old_energy) / kinetic_energy)
             motion.beads.p[:] = alpha * dstrip(motion.beads.p)
-        motion.integrator.pconstraints()
+
```

Comment:
(This one was useless, as `pconstraints()` is also applied before momentum rescaling)

Reply:
I think we had it around when we thought there were numerical instabilities causing momentum drift